metadata.json

```diff
@@ -8,15 +8,15 @@
   "dependencies": [
     {
       "name": "puppetlabs/concat",
-      "version_requirement": ">= 1.0.0"
+      "version_requirement": ">= 1.0.0 <= 5.3.0"
```

Comment:
We can probably loosen these up to less than the next major version, since these are all pretty good about sticking to Semver. Eg `< 6.0.0`